src/client/utils/feature-detection.js

```diff
@@ -2,5 +2,10 @@ import nativeMethods from '../sandbox/native-methods';
 
 // NOTE: In some browsers, elements without the url attribute return the location url
 // when accessing this attribute directly. See form.action in Edge 25 as an example.
-export var emptyActionAttrFallbacksToTheLocation = nativeMethods.createElement.call(document, 'form').action ===
-                                                   window.location.toString();
+export const emptyActionAttrFallbacksToTheLocation = nativeMethods.createElement.call(document, 'form').action ===
+                                                     window.location.toString();
+
+// NOTE: In Chrome, toString(new Blob([])) equals '[object Blob]' and toString(Blob.prototype) equals '[object Blob]',
```

Comment:
// NOTE: In Chrome, toString(new Blob([])) equals '[object Blob]' and toString(Blob.prototype) equals '[object Blob]';
// this condition is also satisfied for Window, Document, XMLHttpRequest, etc